modules/oadp-ssec-encrypted-backups.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-ssec-encrypted-backups_{context}"]
+= OADP SSE-C encrypted backups
+
+Amazon Web Services (AWS) S3 now applies server-side encryption with Amazon S3 managed keys (SSE-S3) as the base level of encryption for every bucket in Amazon S3. It is important to note that OpenShift API for Data Protection (OADP) encrypts data by using SSL/TLS, HTTPS, and the velero-repo-credentials secret when transferring the data off cluster to storage.
+
+It is important to include an additional layer of encryption to prevent backed up data from becoming unencrypted in case of lost or stolen AWS credentials.
```

Comment:
Using backed up data means it is a compound adjective, and should be hyphenated
```suggestion
It is important to include an additional layer of encryption to prevent backed-up data from becoming unencrypted in the event of lost or stolen AWS credentials.
```